fxa-auth-db-server/lib/bufferize.js

```diff
@@ -15,22 +15,38 @@ function unbuffer(object) {
   return object
 }
 
-function bufferize(object, ignore) {
+function bufferize(object, onlyTheseKeys) {
   var keys = Object.keys(object)
+  if (onlyTheseKeys) {
+    keys = keys.filter(key => onlyTheseKeys.has(key))
+  }
   for (var i = 0; i < keys.length; i++) {
     var key = keys[i]
     var value = object[key]
-    if (ignore.indexOf(key) === -1 && typeof value === 'string' && HEX_STRING.test(value)) {
-      object[key] = Buffer(value, 'hex')
+    // Don't convert things with no value, but we still want
+    // to bufferize falsy things like the empty string.
+    if (typeof value !== 'undefined' && value !== null) {
+      if (typeof value !== 'string' || ! HEX_STRING.test(value)) {
+        throw new Error('Invalid hex data for ' + key + ': "' + value + '"')
+      }
+      object[key] = Buffer.from(value, 'hex')
```

Comment:
Yeap, but a custom error is great because it will actually tell us what went wrong